api/game_configurations/lists.yaml

```diff
@@ -0,0 +1,137 @@
+      <br> <br>
+      <ul>
+        <li>Instruction</li> 
+        <ol> Yow will be given a set of integers. Print out sum of integers and finish when flows meets negative integer<br>
+        </ol>
+      </ul>
+    blocks:
+      - variable
+      - text
+      - output
+      - input
+      - controls_if
+      - math_number
+      - math_arithmetic
+      - logic_compare
+      - logic_boolean
+      - controls_forEach
+      - lists_create_with
+      - controls_flow_statements
+  - external_id: list_index
+    title: List with index
+    description: Practice list with index
+    status: active
+    level_number: 1
+    inputs: int
+    num_inputs: 2
+    list: 3
+    list_pos_int: 3
+    prompt: >
+      You can practice basic concept of list<br>
+      <cite> In computer science, many data structures (such as list, an associative array, map, symbol table, or dictionary) 
+      are abstract data types composed of a collection of (key, value) pairs, 
+      such that each possible key appears at most once in the collection. (Wikipedia) </cite>
+      <br>
+      It is very important to understand this concept of (key,value) when you're programming 
+      because you need to frequently search values based on the key or index  <br>
+      <ul>
+        <li>Instruction</li> 
+        <ol> You will be given a set of integers. You need to find out the value of 2nd index</ol>
+      </ul>
+    blocks:
+      - variable
+      - text
+      - output
+      - input
+      - controls_if
+      - math_number
+      - math_arithmetic
+      - lists_indexOf
+      - lists_create_with
+      - lists_getIndex
+      - lists_setIndex
+  - external_id: list_sort
+    title: Sort Lists
+    description:  get the feel of sorting algorithm
+    status: active
+    level_number: 2
+    inputs: int
+    num_inputs: 2
+    prompt: >
+      <cite> In computer science, a sorting algorithm is an algorithm that puts elements of a list in a certain order. <br>
+      This algorithm is made up of a series of instructions that takes an array as input, 
+      performs specified operations on the array, sometimes called a list, and outputs a sorted array. 
+      Sorting algorithms are often taught early in computer science classes 
+      as they provide a straightforward way to introduce other key computer science topics like Big-O notation.(Wikipedia) </cite>
+      <br><br>
+      However, this game will give you a very basic sense of sorting of numbers 
+      when you use a specific block which already implemented with sorting algorithm.<br>
+      <ul>
+        <li>Instruction</li> 
+        <ol> Yow will be given a set of integers. Print out sorted integers<br>
+      </ul>
+    blocks:
+      - variable
+      - text
+      - output
+      - input
+      - controls_if
+      - math_number
+      - lists_length
+      - lists_isEmpty
+      - lists_create_with
+      - lists_sort
+  - external_id: fizz_buzz
```

Comment:
this should probably go in the `loops` module

Reply:
Right.. only reason I put it here.. we have 6 games for looping .. only 4 games for lists.. so If I move it to looping group it will have 7 games for looping and 3 games for lists.. Please advise me on further action..

Reply:
How about moving it to loops and we'll write up a couple of new lists games.

Reply:
How about moving it to loops and we'll write up a couple of new lists games.

Reply:
Sure.. I am working on it.